ansible/files/envoy_config/envoy.service.j2

```diff
@@ -0,0 +1,22 @@
+Description=Envoy Proxy Server
+After=postgrest.service gotrue.service adminapi.service
+Wants=postgrest.service gotrue.service adminapi.service
+
+[Service]
+Type=simple
+#ExecStart=/opt/envoy/envoy --config-path /opt/envoy/envoy.yml
```

Comment:
does this not work?

Reply:
sorry left some testing code in!  the exec start under it does work :)

Reply:
Yeah but why can't we execute envoy directly? We do we need the intermediary bash?